ngraph/frontend/onnx/frontend/src/core/node.cpp

```diff
@@ -80,8 +80,8 @@ class Node::Impl {
     std::vector<std::reference_wrapper<const std::string>> m_output_names;
     mutable std::string m_description;
 
-    bool m_has_subgraph;
-    std::shared_ptr<Subgraph> m_subgraph;
+    bool m_has_subgraphs;
```

Comment:
Just wondering if this could be removed in favor of returning `m_subgraphs.size() > 0` from `has_subgraphs()`